tensorflow/stream_executor/dnn.h

```diff
@@ -879,6 +879,19 @@ string ElementwiseOperationString(ElementwiseOperation op);
 // Suite of operations typically used for implementing Deep/Convolutional Neural
 // Nets. Note: A false return value of an operation indicates the
 // implementation is not available.
+class VersionInfo {
+ public:
+  VersionInfo(int major = 0, int minor = 0, int patch = 0)
+      : major_(major), minor_(minor), patch_(patch) {}
+  int maj() { return major_; }
```

Comment:
use the full name "major" and "minor"

Reply:
Also please make a comment on what doesn't work. So we can root cause in the future. Thanks!

Reply:
Thanks, I renamed the methods as you suggested and added references to issue and an earlier PR with links. Let me know if you prefer a detailed comment